ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/underlay/ebgp/spine-router-bgp.j2

```diff
@@ -19,68 +15,71 @@ router_bgp:
     connected:
       route_map: RM-CONN-2-BGP
 
-{% if underlay_rfc5549 is arista.avd.defined(true) %}
-{# future place to add RFC5549 EBGP peerings #}
-
-{% else %}
 {# Regular EBGP peerings #}
 {# Spine router bgp neighbors configuration #}
+{% if underlay_rfc5549 is arista.avd.defined(true) %}
```

Comment:
Please move the complete `neighbor_interfaces` for rfc5549 into it's own block instead of reusing the logic like this.
Below the leaf<->spine links you have super-spines and overlay-controllers which would produce invalid structured config if set.

Reply:
We will keep it as-is for now and discuss how to refactor afterwards.